tests/tools/test_cli.py

```diff
@@ -139,17 +139,18 @@ def test_get_cli_structure_help(self, mocker, fake_metadata):
         assert isinstance(help_cli_structure, dict)
         assert isinstance(help_cli_structure["kedro"], dict)
 
-        for k, v in help_cli_structure["kedro"].items():
-            assert isinstance(k, str)
-            if isinstance(v, dict):
-                for sub_key in v:
-                    assert isinstance(help_cli_structure["kedro"][k][sub_key], str)
-                    assert help_cli_structure["kedro"][k][sub_key].startswith(
-                        "Usage:  [OPTIONS]"
-                    )
-            elif isinstance(v, str):
-                assert v.startswith("Usage:  [OPTIONS]")
+        self.recursive_cli_structure_test(help_cli_structure["kedro"])
 
         assert sorted(list(help_cli_structure["kedro"])) == sorted(
             DEFAULT_KEDRO_COMMANDS
         )
+
+    def recursive_cli_structure_test(self, structure):
```

Comment:
Nice one! 👍 Maybe just rename it to something like `recursively_check_cli_structure`

Reply:
Renamed in [0b04fa4](https://github.com/kedro-org/kedro/pull/2796/commits/0b04fa477da7f6d62662c7b243a1bf61bbb26943)